paper.md

```diff
@@ -48,15 +48,15 @@ The package provides convenience functions for visualizing spatial patterns and
   - The Approximate Profile Likelihood (APLE) estimator for measuring spatial autocorrelation [@li_2007]
   - Effective sample size (ESS) calculation [@griffith_2005]
 
-These tools are provided for exploratory analysis, not 'cluster detection'; p-values are not provided. Graphics are created with **ggplot2** [@wickham_2016].
+These tools are provided for exploratory analysis, but not for detection of clusters so p-values are not provided. Graphics are created with **ggplot2** [@wickham_2016].
```

Comment:
Can we change:

"These tools are provided for exploratory analysis, but not for detection of clusters so p-values are not provided"

to:

These tools are provided for exploratory analysis, but not for detection of clusters. For this and other reasons, p-values are not provided.

Reply:
@ConnorDonegan - you can first accept it as is and then fix them as you suggested. Please ping me when you've done with it.